-- Use setuptools instead of uv_build as the build backend, because uv_build
-- is not available as a FreeBSD port and the package is pure Python.
-- Also use a PEP 621 compliant license declaration for the current setuptools.

--- pyproject.toml.orig	1970-01-01 00:00:00 UTC
+++ pyproject.toml
@@ -5,7 +5,7 @@ readme = "README.md"
 authors = [{ name = "Maciej Rapacz", email = "mmrapacz@protonmail.ch" }]
 requires-python = ">=3.10,<4"
 readme = "README.md"
-license = "MIT"
+license = {text = "MIT"}
 classifiers = [
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3.10",
@@ -33,5 +33,5 @@ module-root = ""
 module-root = ""
 
 [build-system]
-requires = ["uv_build>=0.8.24,<0.9.0"]
-build-backend = "uv_build"
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
